### PR TITLE
libquicktime: add CVE-2016-2399 patch, simple test

### PR DESCRIPTION
**Pull request by DomT4**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/libquicktime.rb [hunk 1]
```diff
@@ -28,6 +28,14 @@ class Libquicktime < Formula
   end
   patch :DATA
 
+  # Fix CVE-2016-2399. Applied upstream on March 6th 2017.
```
Comment (ilovezfs) [inline Formula/libquicktime.rb]: American date format? This is interesting behavior.

**Review by ilovezfs — APPROVED** [review]

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
:play_or_pause_button:

File: Formula/libquicktime.rb (hunk 1 above)
Comment (DomT4) [inline Formula/libquicktime.rb]: Us Brits tend to use this format in more formal settings as well 😸. The biggest difference I'm aware of between our two dates is the shortest one, 31/10/17 Vs 10/31/17, etc.

Comment (ilovezfs) [inline Formula/libquicktime.rb]: I see, so it bespeaks the gravity of the CVE.

Comment (DomT4) [inline Formula/libquicktime.rb]: Nothing but the best in the name of security, heh.

**Pull request closed without merging**